### DISCUSSION TITLE
NC and grid content not converted?

### DISCUSSION THREAD

**Discussion by skttl** [Jumoo/uSyncMigrations#191] [Q&A] [answered]
I'm dipping my toes in my first migration after uSync Migration got the packer for exporting the old site including config files.

When importing into the new site, and selecting the migration plan for NC to BlockList and Grid to Block List, the migrated content files still have Nested Content looking data, eg.
```xml
<socialMediaIcons>
      <Value><![CDATA[[
  {
    "key": "aba129ff-6b58-41b0-9970-ec03ed92c8c4",
    "name": "http://facebook.com",
    "ncContentTypeAlias": "socialMediaIcons",
    "link": "[{\"name\":\"http://facebook.com\",\"target\":\"_blank\",\"url\":\"http://facebook.com\"}]"
  },
  {
    "key": "a9eb9e13-3dfc-4cc8-b8f6-65e745ffd38d",
    "name": "http://linkedin.com",
    "ncContentTypeAlias": "socialMediaIcons",
    "link": "[{\"name\":\"http://linkedin.com\",\"url\":\"http://linkedin.com\"}]"
  }
]]]></Value>
    </socialMediaIcons>
```

Same thing goes with grid content:

```xml
    <pageContent>
      <Value><![CDATA[{
  "name": "Fuld bredde",
  "sections": [
    {
      "grid": "12",
      "rows": []
    }
  ]
}]]></Value>
    </pageContent>
```

Is this correct?

**Comment by KevinJump** [comment, marked as answer]
Hi, 

Yeah it doesn't look correct. 

the `pageContent` element should contains a BlockList (not a grid). which is what it looks like its done.
(although noted in the top one is 'socialMediaIcons' and in the bottom its a diffrent property ? 

when you run a migration the migrated file will get output into a "Migrated" folder inside the migration folder, and this is where it is actually imported from. I would check that.  to confirm.

**Reply by KevinJump** [reply]
yeah i see now, sorry i read this as before and after not two seperate instances
